### DISCUSSION TITLE
Layout animation not working on ios

### DISCUSSION THREAD

**Discussion by nehatkkhan786** [software-mansion/react-native-reanimated#5503] [Q&A]
Hey guys I have created a layout animation to a button component but its not working on ios in fact no layout animation is working on ios. But on android its working fine. 

Here is my code. 

```js
 <Animated.View entering={FadeInDown.duration(500)} style={{position:'absolute', bottom:50, width:'100%'}}>
          <Pressable
          style={({pressed})=>[styles.btn, pressed ? {opacity:.8} : '' ]}>
            <Text style={{alignSelf:'center', color:'#ffffff', fontSize:hp('2.5%'), fontWeight:'500', textTransform:'uppercase'}}>Buy Ticket</Text>
          </Pressable>
        </Animated.View>

```

**Comment by MatiPl01** [comment]
Hey!

Do you still have this problem or you managed to solve it?

**Reply by nehatkkhan786** [reply]
I am still having this issue. Works fine when hot reloaded but not when on component mounts.

**Reply by MatiPl01** [reply]
Which version of reanimated library do you use? Have you tried upgrading to the latest one? It is weird that layout animations don't work only on iOS.

Do you use old or new architecture?
